Sources/Core/Shared/ComposedOperation.swift

```diff
@@ -18,7 +18,7 @@ public class ComposedOperation<T: Operation>: Operation {
     }
 
     init(operation composed: T) {
-        self.target = composed as? Operation ?? GroupOperation(operations: [composed])
+        self.target = composed ?? GroupOperation(operations: [composed])
```

Comment:
I thought that was what it was in my last pull request, but it gave me an error saying the cast always succeeds.